Dockerfile

```diff
@@ -1,22 +1,37 @@
-FROM php:7.2-fpm
+ARG PHP_IMAGE_TAG=7.2-fpm
+FROM php:${PHP_IMAGE_TAG}
```

Comment:
I wasn't oware of this way of doing things, is that permit to be overridden by a child image ?

Reply:
I didn't understand your question completely. This way can used for building images with different php base images, you can set `--build-arg PHP_IMAGE_TAG=7.3-fpm` and it works.